backends/jwt_remote.go

```diff
@@ -63,6 +65,12 @@ func NewRemoteJWTChecker(authOpts map[string]string, options tokenOptions, versi
 		}
 	}
 
+	if httpMethod, ok := authOpts["jwt_http_method"]; ok {
+		if httpMethod == "POST" || httpMethod == "GET" || httpMethod == "PUT" {
```

Comment:
This is my bad as I never did before, but could I bother you to refactor the backend to use the `http` constants for methods?

Reply:
@fAuernigg I gather you lost interest in this PR: sorry, that's my bad for taking too long. Let me know if there's something you want to add or edit, elsewise I'll apply the same changes to the `http` backend (at the next point I get some time to do so) and merge.

Cheers!

Reply:
Sorry I missed to read your comment.
I will try to add the method to the http backend.